Doc/library/pickle.rst

```diff
@@ -199,7 +199,7 @@ The :mod:`pickle` module provides the following constants:
 
    An integer, the default :ref:`protocol version <pickle-protocols>` used
    for pickling.  May be less than :data:`HIGHEST_PROTOCOL`.  Currently the
-   default protocol is 4, first introduced in Python 3.4 and incompatible
+   default protocol is 5, first introduced in Python 3.4 and incompatible
```

Comment:
Did you check that protocol version 5 was also introduced in Python 3.4?

Finding where it was introduced might also uncover some information about how it is better than protocol 4, to add to What's New?

Reply:
> Did you check that protocol version 5 was also introduced in Python 3.4?
> 
> Finding where it was introduced might also uncover some information about how it is better than protocol 4, to add to What's New?

good point, found here it was added for python 3.8 (commit c879ff2); will pay attention to it to write a propert What's New, thanks! (also sorry for the delay, got some days off, but now I'm getting back to my backlog of contributions :))